hera_mc/cm_utils.py

```diff
@@ -16,6 +16,17 @@
 PAST_DATE = '2000-01-01'
 
 
+class utility_method:
```

Comment:
I think what you're making here is what's generally referred to as a namespace object ([see eg this StackExchange question](https://stackoverflow.com/questions/20828277/what-is-a-namespace-object)). It would be good to re-use the existing [argparse.Namespace](https://docs.python.org/dev/library/argparse.html#argparse.Namespace) class using the standard nomenclature here.